scarb/tests/package.rs

```diff
@@ -1525,3 +1525,143 @@ fn package_with_publish_disabled() {
             [..]Packaged [..] files, [..] ([..] compressed)
         "#});
 }
+
+#[test]
+fn can_include_additional_files() {
+    let t = TempDir::new().unwrap();
+    simple_project()
+        .manifest_package_extra(indoc! {r#"
+            include = ["target/file.txt", "target/some/", "other/file.txt", "other/some"]
+        "#})
+        .build(&t);
+
+    t.child("target/file.txt")
+        .write_str("some file content")
+        .unwrap();
+    t.child("target/some/file.txt")
+        .write_str("some file content")
+        .unwrap();
+    t.child("other/file.txt")
+        .write_str("some file content")
+        .unwrap();
+    t.child("other/some/dir/file.txt")
+        .write_str("some file content")
+        .unwrap();
+    t.child("other/some/dir/other.txt")
+        .write_str("some file content")
+        .unwrap();
+
+    t.child(".gitignore").write_str("target").unwrap();
+    t.child(".scarbignore").write_str("other").unwrap();
+
+    Scarb::quick_snapbox()
+        .arg("package")
+        .arg("--no-metadata")
+        .current_dir(&t)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"
+        [..] Packaging foo v1.0.0 [..]
+        [..] Verifying foo-1.0.0.tar.zst
+        [..] Compiling foo v1.0.0 ([..])
+        [..]  Finished `dev` profile target(s) in [..]
+        [..]  Packaged [..] files, [..] ([..] compressed)
+        "#});
+
+    PackageChecker::assert(&t.child("target/package/foo-1.0.0.tar.zst"))
+        .name_and_version("foo", "1.0.0")
+        .contents(&[
+            "VERSION",
+            "Scarb.orig.toml",
+            "Scarb.toml",
+            "src/lib.cairo",
+            "src/foo.cairo",
+            "other/some/dir/other.txt",
+            "other/some/dir/file.txt",
+            "other/file.txt",
+            "target/some/file.txt",
+            "target/file.txt",
+        ])
+        .file_eq("VERSION", "1")
+        .file_eq_path("Scarb.orig.toml", t.child("Scarb.toml"))
+        .file_eq_path("src/lib.cairo", t.child("src/lib.cairo"))
+        .file_eq_path("src/foo.cairo", t.child("src/foo.cairo"))
+        .file_matches_nl(
```

Comment:
I think we should check the contents of some (at least one) of the included files.

Reply:
It does check contents: 
```
        snapbox::assert_eq_path(expected_path, self.read_file(path));
```
https://docs.rs/snapbox/0.4.17/snapbox/fn.assert_eq_path.html